README.md

```diff
@@ -1,209 +1,177 @@
+                <RightPane>
+                        <ChannelSettings channelUrl={currentChannel} />
+                </RightPane>
+        </Route>
+        </SendBirdProvider>
+)
 ```
-import React, { useEffect, useState } from 'react';
 
+To show a list of channels or change channels based on routes, implement as below:
+
+```javascript
 import {
-  SendBirdProvider,
-  ChannelList,
-  Channel,
-  ChannelSettings,
-} from 'sendbird-uikit';
-import 'sendbird-uikit/dist/index.css';
-
-export default function Chat({ userId, nickname, theme }) {
-  const [showSettings, setShowSettings] = useState(false);
-  const [currentChannelUrl, setCurrentChannelUrl] = useState(null);
-  return (
-    <div style={{ height: '100vh' }}>
-      <SendBirdProvider
-        appId={APP_ID}
-        theme={theme}
-        userId={userId}
-        nickname={nickname}
-      >
-        <div className="sendbird-app__wrap">
-          <div className="sendbird-app__channellist-wrap">
-            <ChannelList
-              onChannelSelect={(channel) => {
-                if (channel && channel.url) {
-                  setCurrentChannelUrl(channel.url);
-                }
-              }}
-            />
-          </div>
-          <div className="sendbird-app__conversation-wrap">
-            <Channel
-              channelUrl={currentChannelUrl}
-              onChatHeaderActionClick={() => { setShowSettings(true); }}
-            />
-          </div>
-        </div>
-        {showSettings && (
-          <div className="sendbird-app__settingspanel-wrap">
-            <ChannelSettings
-              channelUrl={currentChannelUrl}
-              onCloseClick={() => { setShowSettings(false); }}
-            />
-          </div>
-        )}
-      </SendBirdProvider>
-    </div>
-  )
+        SendBirdProvider,
+        ChannelList,
+        Channel,
+} from "sendbird-uikit";
+import "sendbird-uikit/dist/index.css";
+
+const App = () => {
+        <SendBirdProvider appId={appId} userId={userId} ...>
+                <Router>
+                        <Route>
+                                <Sidebar>
+                                        <Channel />
+                                </Sidebar>
+                                <Route id={yourApp/channelUrl1}>
+                                        <MainPanel>
+                                                <Channel channelUrl={yourApp/channelUrl1} />
+                                        </MainPanel>
+                                </Route>
+                                <Route id={yourApp/channelUrl2}>
+                                        <MainPanel>
+                                                <Channel channelUrl={yourApp/channelUrl2} />
+                                        </MainPanel>
+                                </Route>
+                        </Route>
+                        ...
+                </Router>
+        </SendBirdProvider>
 }
 ```
 
-### Color Set List
```

Comment:
@sf-jeff-lim we need the color set list here

Reply:
Updated